pinatapy/__init__.py

```diff
@@ -49,9 +67,9 @@ def get_all_files(directory: str) -> tp.List[str]:
 
         if os.path.isdir(path_to_file):
             all_files: tp.List[str] = get_all_files(path_to_file)
-            files = [("file",(file, open(file, "rb"))) for file in all_files]
+            files = [("file", (dest_folder_name + file.split("/")[-1], open(file, "rb"))) for file in all_files]
         else:
-            files = [("file", open(path_to_file, "rb"))]
+            files = [("file", (dest_folder_name + path_to_file.split("/")[-1], open(path_to_file, "rb")))]
```

Comment:
Are you sure you need "/" here? https://docs.pinata.cloud/pinata-api/pinning/pin-file-or-directory contains an example with no heading "/" for a solo

Reply:
Or is this for the ones, who would like to post one file in a folder?

Reply:
This is meant for to pin a single file (either in the default location of "/" or in a folder)